t/01-categories.t

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env perl
+
+package WebService::SendGrid::Newsletter::Test::Categories;
+
+use strict;
+use warnings;
+
+use lib 't/lib';
+
+use Test::More;
+use Test::Exception;
+use String::Random;
+
+use WebService::SendGrid::Newsletter;
+
+use parent 'WebService::SendGrid::Newsletter::Test::Base';
+
+my $sgn;
+my $category_name;
+my $newsletter_name = 'Test Newsletter';
+
+sub startup : Test(startup => no_plan) {
+    my ($self) = @_;
+
+    $self->SKIP_ALL('SENDGRID_API_USER and SENDGRID_API_KEY are ' .
+        'required to run live tests')
+        unless $ENV{SENDGRID_API_USER} && $ENV{SENDGRID_API_KEY};
+
+    $sgn = WebService::SendGrid::Newsletter->new(
+        api_user => $ENV{SENDGRID_API_USER},
+        api_key  => $ENV{SENDGRID_API_KEY},
+    );
+
+    # Requires an existing newsletter in order to assign recipient to
+    $sgn->add(
+        identity => 'This is my test marketing email',
+        name     => $newsletter_name,
+        subject  => 'Your weekly newsletter',
+        text     => 'Hello, this is your weekly newsletter',
+        html     => '<h1>Hello</h1><p>This is your weekly newsletter</p>'
+    );
+
+    sleep(60);
+
+    my $string_gen = String::Random->new;
```

Comment:
If we only need one random string, there's no need to instantiate String::Random, let's just do:
```perl
use String::Random qw(random_regex);
...
my $category_name = random_regex(...);
```